### DISCUSSION TITLE
ScanSnap IX500 problem with scanning

### DISCUSSION THREAD

**Discussion by budda85** [rocketraman/sane-scan-pdf#51] [Q&A]
Hey,
i try to scan a document with my scanscap ix500
I use this command:
`./scan -d -x "fujitsu:ScanSnap iX500:5999" -r 300 -v -m Lineart --skip-empty-pages -o /home/jan/Dokumente/scan-123.pdf   `

and this is the log:

> Scanning...
scanadf: rounded value of page-height from 279.4 to 279.406
scanadf: rounded value of page-width from 215.9 to 215.893
scanadf: rounded value of br-x from 215.9 to 215.893
scanadf: rounded value of br-y from 279.4 to 279.406
Scanned document /tmp/scan.psGQr0gEIm/scan-0001
Scanned document /tmp/scan.psGQr0gEIm/scan-0002
scanadf: sane_start: Error during device I/O
Error during device I/O
Scanned 2 pages
Found no scans.

no file is saved and there are no files in the tmp folder either.
when i use the build in software from manjaro, the Scanner is working  without any problems.
do you have an idea?

**Comment by rocketraman** [comment]
I think it's an issue with the sane fujitsu driver. I was having similar issues with my S1500 on my Fedora system but I haven't had it happen in a while. At the moment, I am on sane-backends 1.2.1 and sane-frontends 1.0.14. If you are currently on older versions, try to upgrade.

Another thing you can try as a debugging exercise: if the scanner is plugged into a hub, plug it directly into your computer.

**Comment by budda85** [comment]
Hm i have the same version of the backend and frontend and the scanner is connected directly to the pc :-(

**Reply by rocketraman** [reply]
What kernel version are you on?

**Comment by budda85** [comment]
So, now it is working. Don't know why :-)
Found the Script in the AUR Paketmanager and build it. Know this script here is also working.
Maybe a problem with some dependencies.
Thx for your help and the script.

**Reply by rocketraman** [reply]
I had a similar experience. The problem just went away without a good explanation. At this point I'm suspecting a regression to the USB stack in some earlier version of the kernel, and a kernel update fixed it. As a datapoint, I'm currently on sane-backends 1.2.1, sane-frontends 1.0.14, and kernel 6.2.11-300.fc38.x86_64 and it is working fine. Anyway, good to hear its working for you as well.

BTW, anyone else that happens on this, an annoying but effective workaround was to unplug the scanner and replug it. Then usually one or two scans worked before the issue happened again.

If anyone else runs into the issue and figures out the root cause, would love an explanation here!
